utils/celestia/host/src/host.rs

```diff
@@ -171,7 +163,7 @@ impl OPSuccinctHost for CelestiaOPSuccinctHost {
 
 impl CelestiaOPSuccinctHost {
     pub fn new(fetcher: Arc<OPSuccinctDataFetcher>) -> Self {
-        Self { fetcher }
+        Self { fetcher, witness_generator: CelestiaDAWitnessGenerator }
     }
 }
 
```

Comment:
FYI, the latest `hana` has a function for this: https://github.com/celestiaorg/hana/pull/15. Would recommend using that.

Reply:
Made changes in https://github.com/succinctlabs/op-succinct/pull/501/commits/ec03dcb1c00d73248b1e45c6c99d8e51499ee3c0